fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/RuntimeFilterTranslator.java

```diff
@@ -153,7 +158,9 @@ public void createLegacyRuntimeFilter(RuntimeFilter filter, JoinNodeBase node, P
                 //bitmap rf requires isBroadCast=false, it always requires merge filter
                 origFilter.setIsBroadcast(false);
             }
-            boolean isLocalTarget = scanNodeList.stream().allMatch(e -> e.getFragmentId().equals(node.getFragmentId()));
+            boolean isLocalTarget = scanNodeList.stream().allMatch(e ->
+                    e.getStatisticalType() != StatisticalType.CTE_SCAN_NODE
```

Comment:
“e.getStatisticalType() != StatisticalType.CTE_SCAN_NODE” 
is this line necessary?
the fragment ID of a scan node in CTE and a join node outside of CTE are always different, right?